### DISCUSSION TITLE
Listen music from web browser

### DISCUSSION THREAD

**Discussion by alexdrastic** [music-assistant/support#2112] [Feature requests and ideas]
If I use music assistant from my browser and I would listen it form web browser now is not possible. Can you add possibility to listen music from web browser? Thank you

**Comment by OzGav** [comment]
Have you read the docs?

**Comment by calcgenie** [comment]
i have websites. how can i make it as music playing website. give me some ideas

**Reply by OzGav** [reply]
We do NOT advise exposing MA to the Internet. If you are running HA then exposing that and then accessing MA via Ingress (assuming it is installed as an addon) would be an option

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
